jshell.support/src/org/netbeans/modules/jshell/resources/ShellClass.java.template

```diff
@@ -1,3 +1,21 @@
+<#--
```

Comment:
I believe the user will see this with Tools | Templates, then Open in Editor. So, perhaps it should *not* have a license header?